scipy/signal/_spectral_py.py

```diff
@@ -43,11 +57,20 @@ def lombscargle(x,
     y : array_like
         Measurement values.
     freqs : array_like
-        Angular frequencies for output periodogram.
+        Angular frequencies for output periodogram. Frequencies are normally >= 0.
+        Any peak at -freq will also exist at +freq.
     precenter : bool, optional
-        Pre-center measurement values by subtracting the mean.
-    normalize : bool, optional
-        Compute normalized periodogram.
+        Pre-center measurement values by subtracting the mean, if True. This is
+        unnecessary if `floating_mean` is True (default).
+    normalize : bool | str, optional
+        Compute normalized or complex (amplitude + phase) periodogram.
+        Valid options are: ``False``/``"power"``, ``True``/``"normalize"``, or
+        ``"amplitude"``.
+    weights : array_like, optional
+        Weights for each sample. Weights must be nonnegative.
+    floating_mean : bool, optional
+        Determines an offset or each frequency independently, if True.
```

Comment:
Should this be
> an offset _for_ each frequency

?

Reply:
Yup. Not sure how that typo got in there.

Reply:
Fix in https://github.com/scipy/scipy/pull/21697